include/network_interface/network_utils.h

```diff
@@ -105,22 +129,19 @@ T read_be(const std::vector<uint8_t>& bufArray,
   return read_be<T>(bufArray, offset, 1.0, 0);
```

Comment:
what is 1.0 and 0?

Reply:
If you look at the implementation with 4 arguments at line 103 above, they are the `factor` and `offset`. This version of `read_be` only takes 2 arguments (the array and the byte offset of the value to be read) so it assumes you don't want a factor or value offset so it sets them to innocuous values.